README.md

```diff
@@ -6,122 +6,14 @@
 
-#App Generator : http://192.241.232.231
-
-The App Generator is a simple One Pager HTML webpage that allows the user to input details related to the App that he/she wants to be generated.<br>
-These details include :
-
-1. App's Name
-2. API Link from where the json files needed for the App will be fetched
-3. E-Mail address where the generated app will be sent.
-
-The Generator is hosted on a server and is backed up by PHP script which in turn executes python and Bash scripts which start the app's build process. <br>
-After the app is generated,it is emailed to the user using [Mutt](http://www.mutt.org/) and [MSMTP](http://msmtp.sourceforge.net/) <br>
-
-Scripts used in the app generator server can be found [here](https://github.com/fossasia/open-event-android/tree/master/apk-generator/scripts).
-
-#Android client
-This is a generic app that has two parts:
-
-1.A standard configuration file, that sets the details of the app (e.g. color scheme, logo of event, link to JSON app data)<br>
-2.This app uses the json api provided by a server maintained [here](https://github.com/fossasia/open-event-orga-server).
-
-## Development Setup
-Before you begin, you should already have the Android Studio SDK downloaded and set up correctly. You can find a guide on how to do this here: [Setting up Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio)
-
-### Setting up the Android Project
-1. Download the *open-event-android* project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
-
-2. Open Android Studio, you will see a **Welcome to Android** window. Under Quick Start, select *Import Project (Eclipse ADT, Gradle, etc.)*
-
-3. Navigate to the directory where you saved the open-event-android project, select the root folder of the project (the folder named "open-event-android"), and hit OK. Android Studio should now begin building the project with Gradle.
-
-4. Once this process is complete and Android Studio opens, check the Console for any build errors.
-
-	- *Note:* If you recieve a Gradle sync error titled, "failed to find ...", you should click on the link below the error message (if avaliable) that says *Install missing platform(s) and sync project* and allow Android studio to fetch you what is missing.
-
-5. Once all build errors have been resolved, you should be all set to build the app and test it.
-
-6. To Build the app, go to *Build>Make Project* (or alternatively press the Make Project icon in the toolbar).
-
-7. If the app was built succesfully, you can test it by running it on either a real device or an emulated one by going to *Run>Run 'app'* or presing the Run icon in the toolbar.
-
-###Video Walkthrough
-[![FOSSASIA](https://img.youtube.com/vi/n5G4yw3t--U/0.jpg)](https://www.youtube.com/watch?v=n5G4yw3t--U)
-
-### Screenshots  
-![alt-tag](android/screenshots/ss2.PNG)
```

Comment:
Ok, will do it.
But I'll add updated screenshots.
The current ones don't look that good.
So you can go ahead and merge this and later tonight I'll be adding new screenshots